### PR TITLE
typos2

### PR DESCRIPTION
**Pull request by ViPErCZ**
typos

### CODE REVIEW

File: src/Kdyby/RabbitMq/AmqpMember.php [hunk 1]
```diff
@@ -94,10 +94,10 @@
 
 
 	/**
-	 * @param Connection $conn
+	 * @param IConnection $conn
 	 * @param string $consumerTag
 	 */
-	public function __construct(Connection $conn, $consumerTag = null)
+	public function __construct(IConnection $conn, $consumerTag = null)
```
Comment (fprochazka) [inline src/Kdyby/RabbitMq/AmqpMember.php]: That would break the contact of the interface. Why is this needed?

Comment (ViPErCZ) [inline src/Kdyby/RabbitMq/AmqpMember.php]: I need use SSLConnection (https://github.com/php-amqplib/php-amqplib/blob/master/PhpAmqpLib/Connection/AMQPSSLConnection.php) instead of Connection.

**Comment by fprochazka** [comment]
Duplicated by #43
